website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,490 @@
+
+```Jina
+
+{% macro generate_alias_name(custom_alias_name=none, node=none) -%}
+
+    {%- if  env_var('DBT_ENV_TYPE','DEV') == 'DEV' -%}
+
+        {%- if custom_alias_name -%}
+
+            {{ target.schema }}__{{ custom_alias_name | trim }}
+
+        {%- elif node.version -%}
+
+            {{ target.schema }}__{{ node.name ~ "_v" ~ (node.version | replace(".", "_")) }}
+
+        {%- else -%}
+
+            {{ target.schema }}__{{ node.name }}
+
+        {%- endif -%}
+    
+    {%- else -%}
+
+        {%- if custom_alias_name -%}
+
+            {{ custom_alias_name | trim }}
+
+        {%- elif node.version -%}
+
+            {{ return(node.name ~ "_v" ~ (node.version | replace(".", "_"))) }}
+
+        {%- else -%}
+
+            {{ node.name }}
+
+        {%- endif -%}
+
+    {%- endif -%}
+
+{%- endmacro %}
+
+```
+
+```Jina
+
+{% macro generate_schema_name(custom_schema_name, node) -%}
+
+    {%- set default_schema = target.schema -%}
+    {%- if custom_schema_name is none -%}
+
+        {{ default_schema }}
+
+    {%- elif  env_var('DBT_ENV_TYPE','DEV') != 'CI' -%}
+        
+        {{ custom_schema_name | trim }}
+
+    {%- else -%}
+
+        {{ default_schema }}_{{ custom_schema_name | trim }}
+
+    {%- endif -%}
+
+{%- endmacro %}
+
+```
+
+This will generate the following outputs for a model called `my_model` with a custom schema of `marketing`, preventing any overlap of objects between dbt runs from different contexts.
+
+
+| Context     |Target database| Target schema | Resulting object               |
+|-------------|:-------------:|:-------------:|:------------------------------:|
+| Developer 1 | dev           | dbt_dev1      |dev.marketing.dbt_dev1_my_model |
+| Developer 2 | dev           | dbt_dev2      |dev.marketing.dbt_dev2_my_model  |
+| CI PR 123   | ci            | dbt_pr_123    |ci.dbt_pr_123_marketing.my_model|
+| CI PR 234   | ci            | dbt_pr_234    |ci.dbt_pr_234_marketing.my_model|
+| Production  | prod          | analytics     |prod.marketing.my_model         |
+
+
+### 3. Using the branch name for the schema prefix, instead of the developer identity
+
+For teams who prefer to isolate work based on the feature branch, you may want to take advantage of the `DBT_CLOUD_GIT_BRANCH` special environment variable. Please note that developers will write to the exact same schema when they are on the same feature branch.
+
+
+:::note
+
+The `DBT_CLOUD_GIT_BRANCH` variable is only available within the dbt Cloud IDE and not the Cloud CLI.
+
+:::
+
+
+We’ve also seen some organizations prefer to organize their dev databases by branch name. This requires implementing similar logic in `generate_database_name()` instead of the `generate_schema_name()` macro. By default, dbt will not automatically create the databases. You can refer to the [Tips and tricks](/guides/customize-schema-alias?step=4#tips-and-tricks) section on the next page to learn more.    
```

Comment:
fix broken link
```suggestion
We’ve also seen some organizations prefer to organize their dev databases by branch name. This requires implementing similar logic in `generate_database_name()` instead of the `generate_schema_name()` macro. By default, dbt will not automatically create the databases. 

Refer to the [Tips and tricks](p/guides/customize-schema-alias?step=5) section on the next page to learn more.    
```